packages/editor/src/store/reducer.js

```diff
@@ -651,14 +651,18 @@ export function blockSelection( state = {
 				end: action.clientId,
 				initialPosition: action.initialPosition,
 			};
-		case 'INSERT_BLOCKS':
-			return {
-				...state,
-				start: action.blocks[ 0 ].clientId,
-				end: action.blocks[ 0 ].clientId,
-				initialPosition: null,
-				isMultiSelecting: false,
-			};
+		case 'INSERT_BLOCKS': {
+			if ( action.updateSelection ) {
```

Comment:
For me, this is not what's causing the innerBlocks to be selected, it's the `focusTabbable` function inside `block.js`. This on ensures the inserted block (parent) is selected.

Reply:
But the children are inserted after the parent, so without this changes, the selection gets updated on the child blocks are inserted.

Reply:
Oh makes sense. Seems like a decent approach.

Reply:
Hi @youknowriad thank you for looking into this PR. Do you think it is ready to merge?